book/src/user/custom-testnets.md

```diff
@@ -0,0 +1,166 @@
+
+All of these parameters are optional, if they are all omitted or set to their default values, Zebra will run on the default public Testnet.
+
+## Usage
+
+In order to use a custom Testnet, Zebra must be configured to run on Testnet with non-default Testnet parameters. If the node is meant to mine blocks, it will need a `[mining]` section, and if it's meant to mine blocks with non-coinbase transactions, it will also need the `[rpc]` section so the `send_raw_transaction` RPC method is available.
+
+Relevant parts of the configuration file:
+
+```toml
+[mining]
+miner_address = 't27eWDgjFYJGVXmzrXeVjnb5J3uXDM9xH9v'
+    
+[network]
+network = "Testnet"
+
+[network.testnet_parameters.activation_heights]
+network_name = "ConfiguredTestnet_1"
+network_magic = [0, 1, 0, 255]
+slow_start_interval = 0
+target_difficulty_limit = "0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f0f"
+disable_pow = true
+
+# Configured activation heights must be greater than or equal to 1, and less than
+# or equal to Zebra's max block height of 2^31 - 1, block height 0 is reserved
+# for the Genesis network upgrade in Zebra.
+#
+# Network upgrades must be activated in the order that they were added to Zcash,
+# configuring the activation heights of recent network upgrades will activate
+# any omitted prior network upgrades at the same height. 
+#
+# For example, configuring the activation height of NU5 to block height 1 without
+# configuring any other network upgrade activation heights will set the
+# activation heights of BeforeOverwinter, Overwinter, Sapling, Blossom, 
+# Heartwood, and Canopy at block height 1 as well.
+[network.testnet_parameters.activation_heights]
+NU5 = 1
+
+# This section may be omitted if it's not necessary to 
+# add transactions to Zebra's mempool
+[rpc]
+listen_addr = "0.0.0.0:18232"
+```
+
+
+Relevant parts of the configuration file with Mainnet parameters:
+
+```toml
+[mining]
+miner_address = 't27eWDgjFYJGVXmzrXeVjnb5J3uXDM9xH9v'
+    
+[network]
+network = "Testnet"
+
+[network.testnet_parameters.activation_heights]
+# The Mainnet, Testnet, and Regtest network names are reserved.
+network_name = "ConfiguredTestnet_2"
+
+# The Mainnet and Regtest network magics are reserved.
+network_magic = [0, 1, 0, 255]
+
+slow_start_interval = 20_000
+genesis_hash = "00040fe8ec8471911baa1db1266ea15dd06b4a8a5c453883c000b031973dce08"
+
+# Note that setting `disable_pow` to `false` with this target difficultly 
+# limit will make it very difficult to mine valid blocks onto the chain.
+target_difficulty_limit = "0008000000000000000000000000000000000000000000000000000000000000"
+disable_pow = false
+
+[network.testnet_parameters.activation_heights]
+BeforeOverwinter = 1
+Overwinter = 347_500
+Sapling = 419_200
+Blossom = 653_600
+Heartwood = 903_000
+Canopy = 1_046_400
+NU5 = 1_687_104
```

Comment:
I'd like to keep these so that it's possible to copy the Mainnet best chain and checkpoints over to a custom Testnet and disable the Proof-of-Work check to add blocks to it.